modules/load.xqm

```diff
@@ -92,46 +93,200 @@ declare function load:load-archive($name as xs:string, $archive-url as xs:string
+
+
+
+    let $wikidata-play-uri := "http://www.wikidata.org/entity/" ||
       $play//tei:publicationStmt//tei:idno[@type='wikidata']/text()
+
+
+    (:
     let $label := (
       $play//tei:fileDesc/tei:titleStmt//tei:author/text() => string-join(' ')
     ) || ": " || (
       $play//tei:fileDesc/tei:titleStmt//tei:title/text() => string-join(' ')
     )
-    let $dracor-collection := "https://dracor.org/" || $collection-id
-    return
+    :)
+
+    let $collection-uri := "https://dracor.org/" || $collection-id
+
+
+    (: construct rdfs:labels – Author : Title. Subtitle. If there is no @xml:lang on tei:author, skip author-name; but include name in the label in the language of the play :)
+    let $rdfs-labels := for $lang in $titles?*?lang
+        return <rdfs:label xml:lang="{$lang}">{
+            if ( $author-names?*[?lang = $lang] )
+            then $author-names?*[?lang = $lang]?name
+            else if ( not($play//tei:titleStmt//tei:author/@xml:lang) and $play/@xml:lang = $lang ) then $author-names?1?name || ': '
+            else ""
+
+            }
+
+            {
+                if (  $author-names?*[?lang = $lang] and $titles?*[?lang = $lang]?main  ) then ": " else ""
+            }
+            {
+                if ( $titles?*[?lang = $lang]?main ) then $titles?*[?lang = $lang]?main
+                else ""
+            }
+
+            {
+                if ( $titles?*[?lang = $lang]?sub ) then ". " || $titles?*[?lang = $lang]?sub
+                else "."
+            }
+
+            </rdfs:label>
+
+        (: construct dc:creator for each language, if there xml:lang tags on tei:author :)
+            let $dc-creator := if ( count($author-names) > 1 ) then
+                for $creator-lang in $author-names return
+                    <dc:creator xml:lang="{$creator-lang?lang}">{$creator-lang?name}</dc:creator>
+            else
+                <dc:creator>{$author-names?1?name}</dc:creator>
+
+        (: construct dc:title tags for each language :)
+        let $dc-titles := for $lang in $titles?*?lang
+        return
+            if ( $titles?*[?lang = $lang]?main or $titles?*[?lang = $lang]?sub ) then
+
+            <dc:title xml:lang="{$lang}">{
+                if ( $titles?*[?lang = $lang]?main ) then $titles?*[?lang = $lang]?main
+                else ""
+            }
+
+            {
+                if ( $titles?*[?lang = $lang]?sub ) then ". " || $titles?*[?lang = $lang]?sub
+                else "."
+            }</dc:title>
+
+            else ()
+
+        let $in_corpus := <dracon:in_corpus rdf:resource="{$collection-uri}"/>
+        let $play-external-id := <owl:sameAs rdf:resource="{$wikidata-play-uri}"/>
+
+
+    let $inner :=
     <rdf:Description rdf:about="{$play-uri}">
-        <owl:sameAs rdf:resource="{$wikidata}"/>
-        <rdfs:label>{$label}</rdfs:label>
-        <dracon:collection rdf:resource="{$dracor-collection}"/>
+        <owl:sameAs rdf:resource="{$wikidata-play-uri}"/>
```

Comment:
@cmil, #43 I output `owl:sameAs` twice in line 269 and 275 `{play-external-id}`, the first <owl: .. is superfluous

Reply:
I fixed this in 2ed8b52f2dc219d40c2e1fbac5322c5e5b194898.